.env.example

```
@@ -36,3 +36,6 @@ VUE_APP_MIXPANEL_PROJECT_TOKEN= # get this from the Mixpanel project dashboard
 # chatwoot
 VUE_APP_CHATWOOT_URL= # URL of the deployed chatwoot app
 VUE_APP_CHATWOOT_TOKEN= # get this from your deployed chatwoot app's settings
+
+# Sentry Data Source Name
+VUE_APP_SENTRY_DSN=''
```

```suggestion
VUE_APP_SENTRY_DSN=# Sentry Data Source Name
```
Just keeping it consistent with other values! Also if someone copies this file as their `.env` the DSN will be set to an empty string and the error reporting code will run